src/DependencyInjection/MockerContainer.php

```diff
@@ -47,7 +47,7 @@ public function unmock($id)
      *
      * @return object
      */
-    public function get(string $id, int $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE): ?object
+    public function get(/** string */ $id, /** int */ $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE): ?object
```

Comment:
```suggestion
    public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE): ?object
```

As we don't need (I guess) them let's drop them and add them later. Keeping such comments in a codebase seems weird 😣.